generators/server/templates/src/test/java/package/web/rest/AccountResourceIT.java.ejs

```diff
@@ -236,20 +236,6 @@ public class AccountResourceIT <% if (databaseType === 'cassandra') { %>extends
             .jsonPath("$.authorities").isEqualTo(AuthoritiesConstants.ADMIN);
         <%_ } _%>
     }
-
-    @Test
```

Comment:
@Falydoor : should it be excluded only for nodatabase and no oauth2 ?

Reply:
The oauth2 part is in a different `if` and `nodatabase` is kinda linked with `skipUserManagement`.

I tested different scenarios and they were working but let me know if you have one that isn't working!

Reply:
Ok then, trusting you ! Thanks a lot @Falydoor